gems/manageiq_ansible_tower/lib/manageiq_ansible_tower/null_logger.rb

```diff
@@ -0,0 +1,11 @@
+require 'logger'
+
+module ManageiqAnsibleTower
+  class NullLogger < Logger
+    def initialize(*_args)
+    end
+
+    def add(*_args, &_block)
```

Comment:
No need to do the `&_block`...it adds conversion to proc overhead that just isn't needed.

In addition, if you aren't using the args *at all* you may want to just consider `(*)` which is think prevents the args array from even being generated.